src/csharp.tmLanguage.yml

```diff
@@ -2509,6 +2511,18 @@ repository:
       '2': { name: keyword.preprocessor.error.cs }
       '3': { name: string.unquoted.preprocessor.message.cs }
 
+  preprocessor-load:
+    match: \b(load)\b\s*(\"[^"]*\")
```

Comment:
Maybe something like... `\b(load)\b\s*(\"[^"]*[\"$])`

Just a suggestion (assuming it does what I think it will :smile:)